cmd/kubectl-gadget/utils/trace.go

```diff
@@ -185,10 +184,11 @@ func getTraceClient() (*clientset.Clientset, error) {
 	return traceClient, err
 }
 
-func printVersionSkewWarning(pods *corev1.PodList) {
+func PrintVersionSkewWarning(pods *corev1.PodList) {
```

Comment:
is it needed to expose this function?

Reply:
Forgot to roll this back. Exposing this is definitely not needed.